microsoft/pai
# generate an application token for every job

#5270 
- create application token in rest-server, get a k8s secret definition
- create a token object in db, (add a column in table Framework named tokenSecretDef...), let dbc create the secret with the def
- mount the token-secrets to initContainers & job containers
- revoke the token in dbc, (remove the token from DB) : realized in src/dbc

Test cases:
- submit a job with secrets
- check 'kubectl get secret | grep cred', you should see corresponding `configcred` and `tokencred`
- stop the job, the secrets should be deleted automatically
